src/cdomains/apron/linearTwoVarEqualityDomain.apron.ml

```diff
@@ -16,96 +16,113 @@ open VectorMatrix
 
 module Mpqf = SharedFunctions.Mpqf
 
-module Equality = struct
+module Rhs = struct
   (* (Some i, k) represents a sum of a variable with index i and the number k.
      (None, k) represents the number k. *)
   type t = (int option * GobZ.t) [@@deriving eq, ord, hash, show]
   let zero = (None, Z.zero)
   let var_zero i = (Some i, Z.zero)
-  let to_int x = Z.to_int @@ snd x
+  let to_int (_,x) = Z.to_int x
```

Comment:
Of course this was already there before, but still: Maybe `offset_to_int` is a better name for this as it only considers the second component?

Reply:
Since it is not even used, due to us usually deconstructing the tuple anyway, I have removed this strangely named artifact.

I am starting to think, whether modelling the base data as an enumerated data type of either a tuple (int * Z.t) or a single int would have been more appropriate then (int option * Z.t)